### PR TITLE
Replaces the gross language in the beepsky insult

### PR DESCRIPTION
**Pull request by KoboldCommando**
# About The Pull Request

Replaces the insult rant from beepsky and emagged security masks with one that doesn't contain any banned language, leaning into the "radio" gimmick.  Also replaces the text with a line of grawlixes ($#%!).

## Why It's Good For The Game

Consistency with the no sexual content policy.  People will no longer be able to point at Beepsky and say "but it's in the game!"

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
fix: 1984's the Beepsky copypasta
del: Removes Beepsky's ability to break the server rules
/:cl:

### CODE REVIEW

**Review by nednaZ — CHANGES_REQUESTED** [review]
Since both the complionatior and insult files are exactly the same, remove one and just modify the code of the complionator to include the insult.
Saves a bit of file size.

**Comment by Supermichael777** [comment]
Just as a note, we need all ccncsa material for licensing reasons.  We definitely don't have the right to use some of what you sampled for those new files under those licenses.

Those audio scrape bots could pick it up straight off the vods and streams.

Redub a weird rant instead.

**Comment by KoboldCommando** [comment]
Redundant audio file removed, the hailer code now points directly to the beepsky file.  Only 70kb saved, but it adds up!

I'm not really sure if legality is a concern here.  None of the quotes are longer than a second or two, and they all have radio static behind them.  There are several direct quotes that get used on the server and in Ook's stream.  It's ultimately up to Zanden though!

**Comment by Supermichael777** [comment]
if you want to actualy generate new speach, it sucks but this is basicly the only way https://github.com/N3X15/ss13-vox

**Comment by KoboldCommando** [comment]
The bot logic has been changed slightly, now the sound will only play after being emagged, with a 50% chance of the list of emagged sounds being chosen.

This may slightly alleviate copyright concerns, and has the added advantage of making it much easier to toss additional emag sounds in there if anyone makes more!

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 1]
```diff
@@ -251,7 +252,10 @@ Auto Patrol: []"},
 	if(!C.handcuffed)
 		C.handcuffed = new /obj/item/restraints/handcuffs/cable/zipties/used(C)
 		C.update_handcuffed()
-		playsound(src, arrestsounds, 50, 0)//monkestation edit for custom arrest sounds
+		if(EMAGGED && prob(50)) //if it's emagged, there's a chance it'll play a special sound instead
+			playsound(src, emagsounds, 50, 0)
```
Comment (nednaZ) [inline code/modules/mob/living/simple_animal/bot/secbot.dm:256]: This doesn't pick from the list, this is attempting to play the entire list as a sound.

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 2]
```diff
@@ -251,7 +252,10 @@ Auto Patrol: []"},
...
+		else
+			playsound(src, arrestsounds, 50, 0)//monkestation edit for custom arrest sounds
```
Comment (nednaZ) [inline code/modules/mob/living/simple_animal/bot/secbot.dm:258]: As with this

**Review by nednaZ — CHANGES_REQUESTED** [review]
We're not going with that sound effect due to the copyright concerns, a bot will pick up that music on stream instantly.
Confirm we have permission to use the other censored version and use that one instead.

File: code/modules/mob/living/simple_animal/bot/secbot.dm (hunk 1 above)
Comment (KoboldCommando) [reply code/modules/mob/living/simple_animal/bot/secbot.dm:256]: This doesn't seem to be the case for me.  While testing the latest changes I confirmed the hailer, beepsky attack sound, and beepsky arrest sound all pick randomly from their respective lists.

**Comment by KoboldCommando** [comment]
I was told that the soundfile was fine with it being locked behind emagged functions.

I'm still very spooked about taking things from goon, if someone wants to track it down and get permission, the original PR seems to have been from "superlagg" over at goon.  The soundfile list "sec_emag" in code/game/sound.dm is the list picked from for both beepsky and the sec hailer.

Since the main focus here is the removal, I pointed it toward some of the pizzky sound files, which may also serve as a nice hint to people of pizzky's existence.  I chose not to include pizzky's insult.ogg since it's VERY long and VERY loud (on purpose if memory serves).  It also leaves some mystery for the main man himself.

**Review by nednaZ — APPROVED** [review]
Looks good to me.
I like the use of the pizzsky lines

**Pull request merged**